src/Symfony/Component/Validator/Tests/Constraints/ImageValidatorTest.php

```diff
@@ -304,6 +304,18 @@ public function testMaxRatioUsesTwoDecimalsOnly()
         $this->assertNoViolation();
     }
 
+    public function testMaxRatioUsesInputMoreDecimals()
+    {
+        $constraint = new Image([
+            'minRatio' => 4 / 3,
+            'maxRatio' => 4 / 3,
```

Comment:
maxRatio is not needed for the test isn'it?

Reply:
@fancyweb This is testing a fix ration and both should be tested, so the value is exact the same and no rounding issues happening here. 

Maybe best would be a testcase with `4/3` = 1.3333 and `16/9` = 1.7777. What do you think?

Reply:
What I mean if that with the current code, `'maxRatio' => 4 / 3` already pass because `1.33` is not `> 1.333333`.

This test is actually testing `minRatio` for me, that `'minRatio' => 4 / 3` is rounded to `1.33` for the comparison, and that therefore the image is valid. So the test is a bit misleading.

In another test, `'maxRatio' => 16 / 9` would indeed test `maxRatio` properly because the rounded ratio `1.78` is actually `> 1.7777777777778`.

Reply:
A solution for `maxRatio` should be `$ratio > floor($constraint->maxRatio* 100)/100`, this return `true` for `16 / 9` tests, what you think ?

Reply:
The solution of Alexander is already good for me, I was just talking about the tests.

Reply:
@fancyweb updated the tests

/cc @johdcode we should just do the same as we do with the input value here: https://github.com/symfony/symfony/blob/3c51a8ab6f45b0eaefe30b5310d7a8f0d594c3fa/src/Symfony/Component/Validator/Constraints/ImageValidator.php#L165